apps/server/src/controllers/userControllers/authUsers.ts

```diff
@@ -291,3 +299,62 @@ export const forgotPassword = async (req: Request, res: Response) => {
     })
   }
 }
+
+export const resetPassword = async (req: Request, res: Response) => {
+  try {
+    const { verify: longString } = req.query
+
+    const { newPassword } = req.body
```

Comment:
I think the password should be validated using the passwordUtils regex

Reply:
I just updated that. this PR has stayed too long out